Update snapshots.md (#6442)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/snapshots.md

<VersionBlock firstVersion="1.9">

> For information about configure snapshots in dbt versions 1.8 and earlier, select v1.8 from the version picker and it will show up in this section. 

Reviewer: ```suggestion
For information about configuring snapshots in dbt versions 1.8 and earlier, select **1.8** from the documentation version picker, and it will appear in this section.
```

Reviewer: changing some grammar and clarifying its the docs version picker

<VersionBlock firstVersion="1.9">

> For information about configure snapshots in dbt versions 1.8 and earlier, select v1.8 from the version picker or [click here](/docs/build/snapshots?version=1.8). 

Author: ```suggestion
For information about configure snapshots in dbt versions 1.8 and earlier, select v1.8 from the version picker and it will show up in this section. 
```
